### DISCUSSION TITLE
Laravel BP6 Style CSS and script nor working

### DISCUSSION THREAD

**Discussion by DzonyBB** [Laravel-Backpack/community-forum#593] [Q&A (Help)] [answered]
Scripts and stylesheets do not work offline. This is fine ? I don `t understand.

**Comment by jcastroa87** [comment, marked as answer]
Hello @DzonyBB BP6 introduce [basset](https://backpackforlaravel.com/docs/6.x/release-notes#basset-a-better-way-to-load-css-and-js-assets), you need to have internet connection to allow download and generate the local cache.

Cheers.

**Reply by tabacitu** [reply]
Indeed. I would add:
- by default Basset is disabled on local environments; you can disable that in your .ENV file by adding `BASSET_DEV_MODE=false`
- to make sure everything is cached and will work offline, you can do `php artisan basset:cache` (after the above), then everything should work without internet
